README.md

```diff
@@ -1336,6 +1336,7 @@ Software that does not fit in another section.
 
 _See also: [awesome-sysadmin/IT Asset Management](https://github.com/awesome-foss/awesome-sysadmin#it-asset-management)_
 
+- [Actual Budget](https://actualbudget.com) - This is Actual, a local-first personal finance tool. It is now 100% free and open-source. ([Demo](https://app.actualbudget.com/budget), [Source Code](https://github.com/actualbudget/actual-server)) `MIT` `Nodejs/Docker`
```

Comment:
Please change the description to something more meaningful, e.g. the one found on the [website](https://actualbudget.com/)

> Actual is a super fast privacy-focused app for managing your finances.